.goreleaser.yaml

```diff
@@ -68,10 +68,10 @@ builds:
       - -s -w
       - -X "github.com/platformsh/cli/internal/legacy.PHPVersion={{.Env.PHP_VERSION}}"
       - -X "github.com/platformsh/cli/internal/legacy.LegacyCLIVersion={{.Env.LEGACY_CLI_VERSION}}"
-      - -X "github.com/platformsh/cli/commands.version={{.Version}}"
-      - -X "github.com/platformsh/cli/commands.commit={{.Commit}}"
-      - -X "github.com/platformsh/cli/commands.date={{.Date}}"
-      - -X "github.com/platformsh/cli/commands.builtBy=goreleaser"
+      - -X "github.com/platformsh/cli/internal/config.Version={{.Version}}"
```

Comment:
I think we also need to get lines 92-95 below

Reply:
Correct, fixed.